src/app/shared/models/phast/losses/flueGas.ts

```diff
@@ -6,10 +6,7 @@ export interface FlueGas {
 }
 
 export interface FlueGasByMass {
```

Comment:
I am concerned this is going to break existing PHAST assessments

Reply:
Will need to dig deeper

Reply:
Updated commit with these properties kept off. Looking at the history, I introduced them in the first merge and they haven't been in production. Shouldn't be a problem. Shoot me a message to explain more if needed.